tabs.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+function tabs_ds2ad {
+local f="$1"
+
+python3 -c "
+import re
+import sys
+
+def tabs_ds2ad(adt):
+    # Convert tabs from Docusaurus to Asciidoctor
+    replacements = [
+        (r'<Tabs>', ':tabs-sync-option: \n\n[tabs]\n'),
```

Comment:
```suggestion
        (r'<Tabs>', '[tabs]\n====\n'),
```

- A simple replacement to add `:tabs-sync-option:` means pages with multiple tab sets results in `:tabs-sync-option:` added multiple times, but it's a document level attribute. I suggested just removing it completely because it can be enabled at the [site level](https://docs.antora.org/antora/latest/playbook/asciidoc-attributes/#site-attributes).
- Missing the block delimiters (opening and closing ====).